### ISSUE TITLE
where to download latest version anode apk 

### ISSUE THREAD

**Issue by youxiachai** [paddybyers/anode#18]
I just found anode-26-11-11.apk. It is latest version?

**Comment by paddybyers** [comment]
No, there have been lots of changes, but github removed the downloads feature, and so I stopped posting new binaries.

**Comment by gattila** [comment]
And could you upload it to any public site ? For example sourceforge.net ? It can be far more easier to install it from apk instead of compiling from source.
